.github/workflows/test.yml

```diff
@@ -1,14 +1,22 @@
 name: CodeCov
 
-on: [push, pull_request]
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+  push:
+    branches:
+    - master
 
 jobs:
   unit-tests:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ 3.7, 3.8, 3.9 ]  # latest release minus two
+        python-version: [ 3.7, 3.8, 3.9 ]
```

Comment:
django-cms develop supports Python 3.10 now, shouldn't this too?

Reply:
Would need to do that as a separate run. And I'm going to change more of this to run tests from tox because this method of testing is deprecated. This is just a step in the right direction.